test/engine/select.result

```diff
@@ -3721,3 +3721,30 @@ s:fselect(nil, nil, {columns={4, 3, 'name', 'id'}})
   - ​|              "sdf"              |"asdgsdgswegg"|  8  |  7  |
   - ​+---------------------------------+--------------+-----+-----+
 ...
+-- https://github.com/tarantool/tarantool/issues/7040
+-- formatted select (binary data)
+long_bin = '4C6F6E672062696E61727920646174612000000000000000000000000000000080'
+---
+...
+_ = s:update(1, {{'=', 3, '😀'}, {'=', 5, 'non-printable: \x80'}})
+---
+...
+_ = s:replace{3, 4, true, {1, {3, {aa=1,bb='bb\x80bb'}}, 3}}
+---
+...
+_ = s:delete{5}
+---
+...
+_ = s:update(7, {{'=', 'more', string.fromhex(long_bin)}})
+---
+...
+s:fselect({}, nil, {max_width=100})
+---
+- - ​+-----+-----+---------------------------+---------------------------+------------------------+-----+
+  - ​| id  |name |           more            |           col4            |          col5          |col6 |
+  - ​+-----+-----+---------------------------+---------------------------+------------------------+-----+
+  - ​|  1  |  2  |            "😀"            |             4             |Im5vbi1wcmludGFibGU6IIAi|  5  |
+  - ​|  3  |  4  |           true            |WzEsWzMseyJhYSI6MSwiYmIiOiJ|                        |     |
+  - ​|  7  |  8  |IkxvbmcgYmluYXJ5IGRhdGEgXHU|           "sdf"           |   "dsgfsdgsegasges"    |     |
```

Comment:
IMO dumping binary data encoded in base64 without any specification looks confusing. Maybe, add `!!binary` tag, as we do in Console/YaML? Come to think of it, is there any point altogether in printing binary data to the console? Maybe, just print something like `<binary>` instead?

Reply:
Done